app/email.go

```diff
@@ -101,8 +101,8 @@ func SendVerifyEmail(userId, userEmail, locale, siteURL string) *model.AppError
 func SendSignInChangeEmail(email, method, locale, siteURL string) *model.AppError {
 	T := utils.GetUserTranslations(locale)
 
-	subject := T("api.templates.singin_change_email.subject",
-		map[string]interface{}{"SiteName": utils.ClientCfg["SiteName"]})
+	subject := fmt.Sprintf("[%v] %v", utils.Cfg.TeamSettings.SiteName, T("api.templates.singin_change_email.subject",
```

Comment:
I might be missing something but why isn't "[%v] %v" part of the localization string vs doing a `fmt.Sprintf`?

Maybe @enahum can comment?